### PR TITLE
KAA-1002 Change Objective-C SDK type from static library to framework

### PR DESCRIPTION
**Pull request by sergiyk0**
[Objective-C SDK] Change SDK type from static library to framework

### CODE REVIEW

File: client/client-multi/client-objective-c/Kaa/Kaa.h [hunk 1]
```diff
@@ -1,114 +1,140 @@
...
-
-@end
\ No newline at end of file
+//
+//  Kaa.h
+//  Kaa
+//
+//  Created by Serhii Shapoval on 4/8/16.
+//  Copyright © 2016 CYBERVISION INC. All rights reserved.
+//
```
Comment (abohomol) [inline client/client-multi/client-objective-c/Kaa/Kaa.h]: Incorrect copyright header, please use the one that is used in other source files.

File: client/client-multi/client-objective-c/Kaa/Kaa.h [hunk 2]
```diff
@@ -1,114 +1,140 @@
...
+
+#import <UIKit/UIKit.h>
+
+//! Project version number for Kaa.
+FOUNDATION_EXPORT double KaaVersionNumber;
+
+//! Project version string for Kaa.
+FOUNDATION_EXPORT const unsigned char KaaVersionString[];
+
+// In this header, you should import all the public headers of your framework using statements like #import <Kaa/PublicHeader.h>
```
Comment (abohomol) [inline client/client-multi/client-objective-c/Kaa/Kaa.h]: Was that auto generated comment? I'd remove it.

**Comment by abohomol** [comment]
Minor issues mentioned, otherwise looks good to me.

File: client/client-multi/client-objective-c/Kaa/KaaClientFactory.h [hunk 3]
```diff
@@ -0,0 +1,44 @@
+/**
```
Comment (rasendubi) [inline client/client-multi/client-objective-c/Kaa/KaaClientFactory.h:1]: This should be a normal comment, not documentation one.

Comment (abohomol) [inline client/client-multi/client-objective-c/Kaa/KaaClientFactory.h:1]: I suppose it's the one that license plugin provides to us.

**Pull request closed without merging**